src/controllers/messages.controller.ts

```diff
@@ -64,21 +120,42 @@ export const createMessage: RequestHandler = async (req, res) => {
             id: createMessageBody.message.id ? UUIDV4() : null,
         });
 
-        const message = await messagesDB.findOne({ where: { name: createMessageBody.message.id } }).then((message) => message?.toJSON()) as Message;
+        const message = await messagesDB.findOne({ where: { name: createMessageBody.message.id, type: 'MESSAGE' } }).then((message) => message?.toJSON()) as Message;
 
         res.status(201).json({ message });
     } catch (e) {
         res.status(500).json({ message: e });
     }
 };
 
+interface CreateMeasurementBody {
+    measurement:Partial<Measurement>
+}
+
+export const createMeasurement: RequestHandler = async (req, res) => {
```

Comment:
We should meet to discuss message/measurement creation (its a special topic. Lets omit of any message creation routes for this PR